build.gradle

```diff
@@ -61,6 +61,7 @@ subprojects {
             task testSourcesJar(type: Jar, dependsOn: testClasses) {
                 classifier = 'test-sources'
                 from sourceSets.test.allSource
+                file('bin/openwhisk-admin-tools-1.0.0-SNAPSHOT.jar')
```

Comment:
Doesn’t seem right - what’s the problem you’re trying to address? I can’t imagine a downstream dependence on wskadmin-next; what am I missing?